float64 init_radius
---
bool result
